cmd/operator/main.go

```diff
@@ -156,6 +156,7 @@ func init() {
 	flagset.StringVar(&cfg.ReloaderConfig.CPULimit, "config-reloader-cpu-limit", defaultReloaderCPU, "Config Reloader CPU limit. Value \"0\" disables it and causes no limit to be configured. Flag overrides `--config-reloader-cpu` for the CPU limit")
 	flagset.StringVar(&cfg.ReloaderConfig.MemoryRequest, "config-reloader-memory-request", defaultReloaderMemory, "Config Reloader Memory request. Value \"0\" disables it and causes no request to be configured. Flag overrides `--config-reloader-memory` for the memory request")
 	flagset.StringVar(&cfg.ReloaderConfig.MemoryLimit, "config-reloader-memory-limit", defaultReloaderMemory, "Config Reloader Memory limit. Value \"0\" disables it and causes no limit to be configured. Flag overrides `--config-reloader-memory` for the memory limit")
+	flagset.BoolVar(&cfg.ReloaderConfig.IsProbes, "enable-config-reloader-probes", false, "Config Reloader Open LivenessProbe and ReadinessProbe. Default: false")
```

Comment:
How about renaming `IsProbes` to `EnableProbes`? 

```suggestion
	flagset.BoolVar(&cfg.ReloaderConfig.EnableProbes, "enable-config-reloader-probes", false, "Config Reloader Open LivenessProbe and ReadinessProbe. Default: false")
```

Reply:
change `IsProbes` to `EnableProbes `